internal/antewrapper/provenance_fee.go

```diff
@@ -106,16 +107,35 @@ func (dfd ProvenanceDeductFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, s
 	return next(ctx, tx, simulate)
 }
 
-// DeductFees deducts fees from the given account.
-func DeductFees(bankKeeper types.BankKeeper, ctx sdk.Context, acc types.AccountI, fees sdk.Coins) error {
-	if !fees.IsValid() {
-		return sdkerrors.Wrapf(sdkerrors.ErrInsufficientFee, "invalid fee amount: %q", fees)
+func CalculateBaseFee(ctx sdk.Context, feeTx sdk.FeeTx, msgfeekeeper msgfeestypes.MsgFeesKeeper) sdk.Coins {
+	if shouldIgnoreChecksForTests(ctx) {
```

Comment:
Is this backwards? I feel like, if we're ignoring checks for tests, we'd want to return `DetermineTestBaseFeeAmount`.

Reply:
Ok, renamed to something more understandable.